app/src/main/res/layout/playback_speed_feed_setting_dialog.xml

```diff
@@ -34,10 +34,38 @@
 
     </LinearLayout>
 
-    <CheckBox
-        android:id="@+id/skipSilenceFeed"
+    <com.google.android.material.button.MaterialButtonToggleGroup
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/skipSilence"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:text="@string/pref_skip_silence_title" />
+        android:weightSum="3"
+        app:singleSelection="true">
+
+        <Button
+            android:id="@+id/skipSilenceOff"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="1"
+            android:text="Off"
+            style="@style/OutlinedButtonBetterContrast" />
+
+        <Button
+            android:id="@+id/skipSilenceMedium"
```

Comment:
Please add the `button` suffix, similar for the other buttons. See also the code style guidelines: https://github.com/AntennaPod/AntennaPod/wiki/Code-style

```suggestion
            android:id="@+id/skipSilenceMediumButton"
```